scripts/gcp_setup/setup_forseti.py

```diff
@@ -33,12 +33,24 @@ def run():
                         help='Bypass IAM check for user running script')
     parser.add_argument('--branch',
                         help='Which Forseti branch to deploy')
+    parser.add_argument('--unattended-install',
```

Comment:
Can you please explain why we are adding this unattended install option now?  Did anyone specifically ask for it?   The reason I am asking is because now we will have to maintain & test 2 different flows here.  So, I really want to make sure there is a real benefit to add this.